packages/shopify-codemod/transforms/remove-return-voids.js

```diff
@@ -0,0 +1,34 @@
+import {matchLast} from './utils';
+
+export default function removeReturnVoids({source}, {jscodeshift: j}, {printOptions = {}}) {
+
+  return j(source)
+    .find(j.Function, {
+      body: {
+        body: matchLast(j, {
+          type: 'IfStatement',
+          alternate: {
+            body: matchLast(j, {
+              type: 'ReturnStatement',
```

Comment:
You can probably move the filter block to just be a check on the `argument` property here.